bot/cogs/sudo.py

```diff
@@ -32,6 +40,13 @@ def __init__(self, bot: Bot) -> None:
         self.bot = bot
         self.start_time = datetime.datetime.utcnow()
 
+    def cog_check(self, ctx: Context) -> t.Optional[bool]:
+        if ctx.author.id in devs:
+            return True
+        else:
+            embed = Embed(description="Make your Own sandwich Mortal!.", color=Color.red())
+            await ctx.send(embed=embed)
```

Comment:
We don't need to use `else` since when the first if passes it will go directly to `return True` ending the function, only if it doesn't go there it would always go to `else` which means we might as well use flatter code (which is better according to the zen of python) instead.

We should also return `False` in case the check isn't passed
```suggestion
        if ctx.author.id in devs:
            return True
 
        embed = Embed(description="Make your Own sandwich Mortal!.", color=Color.red())
        await ctx.send(embed=embed)
        return False
```